### ISSUE TITLE
PostGIS images do not have the PostGIS extension enabled

### ISSUE THREAD

**Issue by fritzdj** [CircleCI-Public/cimg-postgres#79]
When using the 14.6-postgis image for example, we are seeing the following error when attempting to use the PostGIS extension:
```
ERROR:  function postgis_version() does not exist at character 8
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
STATEMENT:  SELECT PostGIS_version();
```
We didn't realize at first that `CREATE EXTENSION postgis;` needs to run in order to use the extension. Is this expected or is this something that should be included as part of the image OOTB (i.e. how [postgis/postgis](https://registry.hub.docker.com/r/postgis/postgis/) handles it)?

**Comment by JalexChen** [comment]
hi @fritzdj - currently yes, this is expected. we provide the variant with the postgis packages installed, but you would still need to initialize and create the extension within the database itself.

**Comment by JalexChen** [comment]
hi @fritzdj - i shouldn't have closed this out, but there is a PR #87 to enable the postgis extension out of the box.
